src/picongpu/include/plugins/SliceFieldPrinterMulti.tpp

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2013-2014 Heiko Burau, Rene Widera, Felix Schmitt,
+ * Copyright 2013 Heiko Burau, Rene Widera, Felix Schmitt,
```

Comment:
@PrometheusPi that comment is a bit confusing.
he updated the file. so the line `2013-2014` that he falsely removed should read `2013-2015`.